hash/megadriv.xml

```diff
@@ -26649,9 +26699,12 @@ Black screen, writes in ROM area then jumps to PC=1a2000 (encryption?)
 	</software>
 
 	<software name="sonic2b" cloneof="sonic2">
-		<description>Sonic The Hedgehog 2 (World, rev. 01a)</description>
+		<description>Sonic The Hedgehog 2 (World, rev. A, ripped from Sonic Mega Collection)</description>
 		<year>1992</year>
 		<publisher>Sega</publisher>
+		<notes><![CDATA[
+Official modification of rev. A which fixes a bug in the sound test.
+]]></notes>
```

Comment:
Any specifics about what this bug is?

Reply:
All the specifics: https://tcrf.net/Sonic_the_Hedgehog_2_(Genesis)/Revision_Differences#Sonic_Mega_Collection
> The version included in Sonic Mega Collection is the REV01 version (oddly instead of the REV02 version), however it now fixes a bug where the 14 continues cheat (1, 1, 2, 4 on the options menu) causes the Oil Ocean theme to play continually. This is because, despite what the sound test says, the sound IDs actually start at $80 internally, so the 2 player results would be $81 and so on. It's possible that the person programming the cheat mistook the continue jingle's sound test ID ($3F) with its actual ID ($BF), which causes the sound driver to malfunction and be unable to play any new sounds.

Reply:
I've made the comment more specific.